Makefile

```diff
@@ -147,7 +151,7 @@ verify: lint toc-verify
 
 .PHONY: build
 build: manifests fmt vet ## Build manager binary.
-	go build -o bin/manager cmd/main.go
+	go build -ldflags="$(LD_FLAGS) "-o bin/manager cmd/main.go
```

Comment:
we need to make the dockerfile call this build rule, currently it doesn't; see https://github.com/kubernetes-sigs/jobset/pull/780; we can do the exact same thing.